protected/components/views/patientDiagnosesAndMedicationsWidget.php

```diff
@@ -16,13 +16,15 @@
                             <thead>
                             <tr>
                                 <th>Diagnosis</th>
+                                <th>Confirmed/Unconfirmed</th>
                                 <th>Date</th>
                             </tr>
                             </thead>
                             <tbody>
                             <?php foreach ($this->patient->secondarydiagnoses as $diagnosis): ?>
                                 <tr>
                                     <td><?php echo $diagnosis->disorder->fully_specified_name; ?></td>
+                                    <td><?php echo ($diagnosis->is_confirmed) ? 'Confirmed' : 'Unconfirmed'; ?></td>
```

Comment:
This comparison isn't quite right. "is_confirmed" is 1 if the diagnosis is confirmed, 0 if the diagnosis isn't confirmed, and null if it doesn't need to be confirmed. So you should probably used "!== 0" instead of a boolean cast